toponetx/classes/reportviews.py

```diff
@@ -828,3 +826,95 @@ def __contains__(self, e) -> bool:
                 return frozenset(e) in self.nodes
         else:
             return False
+
+
+class PathView(SimplexView):
+    """
+    Path view class.
+
+    Parameters
+    ----------
+    name: str, optional
+        Name of the PathView instance.
+    """
+
+    def __init__(self, name: str = "") -> None:
+        super().__init__(
+            name
+        )  # self.faces_dict is a list of dictionary of allowed paths
```

Comment:
The comment doesn't seem to belong here?

Reply:
It was a comment when I first wrote the code so I didn't forget what should be in `self.faces_dict`. Anyway I just removed it :)